src/inputs/Checkbox.tsx

```diff
@@ -6,17 +6,12 @@ import { CheckboxBase } from "src/inputs/CheckboxBase";
 export interface CheckboxProps {
   label: string;
   checkboxOnly?: boolean;
+  selected: boolean | "indeterminate";
```

Comment:
So, dunno, I always thought it was kind of weird that "indeterminate" was considered orthogonal to selected, b/c afaiu a checkbox really can be only 1 of "selected yes", "selected no", or "indeterminate".

So I nudged it that way, by removing the `indeterminate` prop and instead having callers pass `selected="indeterminate"` to highlight it doesn't really mean anything to be "selected _and_ indeterminate".

Reply:
Doesn't really sit right with me to have a `boolean | "someString"` type but it doesn't appear that an Enum actually simplifies anything.

Reply:
Ah yeah, I admit "mixing separate primitive types in a single union type" is maybe slightly esoteric functional programming sort of a thing, but agreed it's net/net what an enum would be, but generally same (if not more) simplicity, especially for the ~90% case of "it is really just either true/false".